AmazonChimeSDK/AmazonChimeSDKTests/internal/audio/DefaultAudioClientControllerTests.swift

```diff
@@ -90,6 +90,38 @@ class DefaultAudioClientControllerTests: CommonTestCase {
         verify(audioLockMock.lock()).wasCalled()
         verify(audioLockMock.unlock()).wasCalled()
     }
+    
+    func testStart_emptyAudioHostUrl() {
+        DefaultAudioClientController.state = .stopped
+        given(audioSessionMock.getRecordPermission()).willReturn(.granted)
+
+        XCTAssertThrowsError(try defaultAudioClientController.start(audioFallbackUrl: audioFallbackUrl,
```

Comment:
no way to check the specific error thrown?

Reply:
Good point! let me check.

Reply:
Updated